pytorch/training/docker/1.10/py3/cu113/Dockerfile.sagemaker.gpu

```diff
@@ -21,7 +21,7 @@ ENV SAGEMAKER_TRAINING_MODULE=sagemaker_pytorch_container.training:main
 
 # swap the pytorch training wheel with additional smdebug and smmdp features
 ARG PT_TRAINING_URL=https://aws-pytorch-cicd-v3-binaries.s3.us-west-2.amazonaws.com/r1.10.2_aws_v3/20220209-025350/bea1183a47174ef43ab9874c643357fa77755932/gpu/torch-1.10.2%2Bcu113-cp38-cp38-manylinux1_x86_64.whl
-ARG SMD_MODEL_PARALLEL_URL=https://sagemaker-distributed-model-parallel.s3.us-west-2.amazonaws.com/pytorch-1.10.0/build-artifacts/2022-02-21-19-26/smdistributed_modelparallel-1.7.0-cp38-cp38-linux_x86_64.whl
+ARG SMD_MODEL_PARALLEL_URL=https://sagemaker-distributed-model-parallel.s3.us-west-2.amazonaws.com/pytorch-1.10.0/build-artifacts/2022-04-14-03-58/smdistributed_modelparallel-1.8.1-cp38-cp38-linux_x86_64.whl
```

Comment:
@YangFei1990 is there a reason this is versioned to 1.10.0? Also, does this release require re-release of HF images as well?

Reply:
There are bug fixes and features that required by customer for PT1.10. [HF DLC PR](https://github.com/aws/deep-learning-containers/pull/1800) has been merged already.